scipy/stats/_stats_py.py

```diff
@@ -7501,48 +7501,84 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            in the case of a correlated system of variables is such that it can be reasonably
            supposed to have arisen from random sampling", Philosophical Magazine. Series 5. 50
            (1900), pp. 157-175.
+    .. [4] Mannan, R. William and E. Charles. Meslow. “Bird populations and
+           vegetation characteristics in managed and old-growth forests,
+           northeastern Oregon.” Journal of Wildlife Management
+           48, 1219-1238, :doi:`10.2307/3801783`, 1984.
 
     Examples
     --------
-    When just `f_obs` is given, it is assumed that the expected frequencies
-    are uniform and given by the mean of the observed frequencies.
+    In [4]_ bird foraging behavior was investigated in a forest of Oregon.
+    The foraging refers to the range of activities and behaviours exhibited by
+    birds in their quest for food.
```

Comment:
```suggestion
    In [4]_, bird foraging behavior was investigated in a forest of Oregon.
```
If this is a technical definition of foraging, it is not so different from the colloquial definition that it needs to be defined. I'd leave it out.

Or maybe instead of defining "foraging", you want to define _"foraging event"_ so you can use the term below?